components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -241,7 +241,7 @@ so they are not relevant when assigning IAM resources such as teams or roles.
 
 #### Configuring Project Limit
 
-By default, Chef Automate is currently limited to six projects. If you would like to increase that limit, you will need to do so using the Chef Automate CLI:
+By default, Chef Automate limits to six projects. Use the Chef Automate CLI to increase this project limit.
```

Comment:
That seems to be missing an object; how about "limits you"...?
(Same comment applies in the iam-v2-overview.md file below.)

Reply:
```suggestion
By default, Chef Automate limits you to six projects. You can increase the project limit using the CLI: 
```